Uyanda.Coffee.Persistence/LocalDbContext.cs

```diff
@@ -32,8 +34,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             {
                 entity.Property(p => p.Name).HasColumnType("varchar(128)");
 
-                entity.ToTable("Beverage");
```

Comment:
While this might be counter intuitive at first. . . We tend to keep table name singular. You can Google the Pros and Cons of Singular vs. Plural